### ISSUE TITLE
[PATCH] Update Math::BigInt to CPAN version 1.993

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#11154]
Migrated from [rt.perl.org#84844](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=84844) (status was 'resolved')

Searchable as RT84844$

**Comment by p5pRT** [comment]
### From @pjacklam

Change default backend library from Math&#8203;::BigInt&#8203;::FastCalc to
Math&#8203;::BigInt&#8203;::Calc\, which is included in the Math\-BigInt distro\.
This avoids recursive distribution dependency \(RT \#65976\)\.


---
 dist/Math\-BigInt/lib/Math/BigFloat\.pm       |    4 \+\+\-\-
 dist/Math\-BigInt/lib/Math/BigInt\.pm         |    8 \+\+\+\+\-\-\-\-
 dist/Math\-BigInt/lib/Math/BigInt/Calc\.pm    |    2 \+\-
 dist/Math\-BigInt/lib/Math/BigInt/CalcEmu\.pm |    2 \+\-
 4 files changed\, 8 insertions\(\+\)\, 8 deletions\(\-\)

<details><summary>Inline Patch</summary>

```diff
diff --git a/dist/Math-BigInt/lib/Math/BigFloat.pm b/dist/Math-BigInt/lib/Math/BigFloat.pm
index a39d786..07d33cc 100644
--- a/dist/Math-BigInt/lib/Math/BigFloat.pm
+++ b/dist/Math-BigInt/lib/Math/BigFloat.pm
@@ -12,7 +12,7 @@ package Math::BigFloat;
 #   _a	: accuracy
 #   _p	: precision
 
-$VERSION = '1.992';
+$VERSION = '1.993';
 require 5.006002;
 
 require Exporter;
@@ -60,7 +60,7 @@ $upgrade = undef;
 $downgrade = undef;
 # the package we are using for our private parts, defaults to:
 # Math::BigInt->config()->{lib}
-my $MBI = 'Math::BigInt::FastCalc';
+my $MBI = 'Math::BigInt::Calc';
 
 # are NaNs ok? (otherwise it dies when encountering an NaN) set w/ config()
 $_trap_nan = 0;
diff --git a/dist/Math-BigInt/lib/Math/BigInt.pm b/dist/Math-BigInt/lib/Math/BigInt.pm
index a9de794..3e22d0f 100644
--- a/dist/Math-BigInt/lib/Math/BigInt.pm
+++ b/dist/Math-BigInt/lib/Math/BigInt.pm
@@ -18,7 +18,7 @@ package Math::BigInt;
 my $class = "Math::BigInt";
 use 5.006002;
 
-$VERSION = '1.992';
+$VERSION = '1.993';
 
 @ISA = qw(Exporter);
 @EXPORT_OK = qw(objectify bgcd blcm); 
@@ -172,8 +172,8 @@ $_trap_nan = 0;				# are NaNs ok? set w/ config()
 $_trap_inf = 0;				# are infs ok? set w/ config()
 my $nan = 'NaN'; 			# constants for easier life
 
-my $CALC = 'Math::BigInt::FastCalc';	# module to do the low level math
-					# default is FastCalc.pm
+my $CALC = 'Math::BigInt::Calc';	# module to do the low level math
+					# default is Calc.pm
 my $IMPORT = 0;				# was import() called yet?
 					# used to make require work
 my %WARN;				# warn only once for low-level libs
@@ -2742,7 +2742,7 @@ sub import
     {
     $_ =~ tr/a-zA-Z0-9://cd;			# limit to sane characters
     }
-  push @c, \'FastCalc', \'Calc'			# if all fail, try these
+  push @c, \'Calc'				# if all fail, try these
     if $warn_or_die < 2;			# but not for "only"
   $CALC = '';					# signal error
   foreach my $l (@c)
diff --git a/dist/Math-BigInt/lib/Math/BigInt/Calc.pm b/dist/Math-BigInt/lib/Math/BigInt/Calc.pm
index a84786e..25f9a3b 100644
--- a/dist/Math-BigInt/lib/Math/BigInt/Calc.pm
+++ b/dist/Math-BigInt/lib/Math/BigInt/Calc.pm
@@ -4,7 +4,7 @@ use 5.006002;
 use strict;
 # use warnings;	# dont use warnings for older Perls
 
-our $VERSION = '1.992';
+our $VERSION = '1.993';
 
 # Package to store unsigned big integers in decimal and do math with them
 
diff --git a/dist/Math-BigInt/lib/Math/BigInt/CalcEmu.pm b/dist/Math-BigInt/lib/Math/BigInt/CalcEmu.pm
index 3f2aa3b..ee0b677 100644
--- a/dist/Math-BigInt/lib/Math/BigInt/CalcEmu.pm
+++ b/dist/Math-BigInt/lib/Math/BigInt/CalcEmu.pm
@@ -5,7 +5,7 @@ use strict;
 # use warnings;	# dont use warnings for older Perls
 use vars qw/$VERSION/;
 
-$VERSION = '1.992';
+$VERSION = '1.993';
 
 package Math::BigInt;
 
-- 
1.7.4
```
</details>

**Comment by p5pRT** [comment]
### From @bingos

On Thu\, Feb 24\, 2011 at 04&#8203;:52&#8203;:17AM \-0800\, Peter J\. Acklam via RT wrote&#8203;:
> \# New Ticket Created by  \(Peter J\. Acklam\) 
> \# Please include the string&#8203;:  \[perl \#84844\]
> \# in the subject line of all future correspondence about this issue\. 
> \# \<URL&#8203;: http&#8203;://rt\.perl\.org/rt3/Ticket/Display\.html?id=84844 >
> 
> 
> Change default backend library from Math&#8203;::BigInt&#8203;::FastCalc to
> Math&#8203;::BigInt&#8203;::Calc\, which is included in the Math\-BigInt distro\.
> This avoids recursive distribution dependency \(RT \#65976\)\.


Thanks\, applied as 

a90064ab37740661fd8275f18ad2a5cf677473ec

\-\- 
Chris Williams
aka BinGOs
PGP ID 0x4658671F
http&#8203;://www\.gumbynet\.org\.uk
==========================

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
@cpansprout - Status changed from 'open' to 'resolved'
